test/integration/api/users.js

```diff
@@ -287,6 +287,22 @@ describe('api: /users', () => {
               .expect(401);
           }))));
 
+    it('should return 403 if user doesn\'t have rights to invalidate', testService((service) =>
```

Comment:
How about we update this test description to make it clear that it's different from the similar test above (the test that "should fail the request if invalidation is requested but not allowed"). Maybe something like "should return 403 if user doesn't have rights to invalidate and email doesn't exist" or "should fail the request if invalidation is not allowed and email doesn't exist".